sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcSourceSuite.scala

```diff
@@ -62,6 +63,22 @@ class HiveOrcSourceSuite extends OrcSuite with TestHiveSingleton {
        """.stripMargin)
   }
 
+  test("SPARK-22972: hive orc source") {
+    spark.sql(
+      s"""CREATE TABLE normal_orc_as_source_hive
```

Comment:
Put it next line. You can refer the other test cases of `CREATE TABLE `

Reply:
done